data/js/menu.js

```diff
@@ -21,8 +21,8 @@ menu.data = [{
     "title": "Settings",
     "hyperRef": "/settings.html"
 }, {
-    "title": "Configuration Backup",
-    "hyperRef": "/config_backup.html"
+    "title": "Update",

```

Comment:
Backup and Update are very different operations. 

Are you replacing Backup with Update ? Or should this become "Backup and Update" ?